airqo-platform/AirQo-frontend
# [MOBILE] Fix bugs in CI/CD for android production

#### Summary of Changes (What does this PR do?)

- Resolves CI/CD issues for android production track

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
- To test locally, update the app version, cd into the android directory, and run `bundle exec fastlane production` ensure to have the playstore service account in your environment variables
- The github action can only be tested after the PR has been merged

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
![Screenshot 2023-04-01 at 13 58 59](https://user-images.githubusercontent.com/60974514/229287601-2162f950-4139-4a52-b8d2-264a26526497.jpeg)
